engine/gamesys/src/gamesys/scripts/script_tilemap.cpp

```diff
@@ -278,8 +309,8 @@ namespace dmGameSystem
             set_hull_ddf.m_Column = cell_x;
             set_hull_ddf.m_Row = cell_y;
             set_hull_ddf.m_Hull = tile;
-            set_hull_ddf.m_FlipHorizontal = flip_h;
-            set_hull_ddf.m_FlipVertical = flip_v;
+            set_hull_ddf.m_FlipHorizontal = bitmask & FLIP_HORIZONTAL;
+            set_hull_ddf.m_FlipVertical = bitmask & FLIP_VERTICAL;
```

Comment:
This is indeed interesting, and I'll admit I completely forgot about the physics part.
Have you tested with physics?
We need to support rotation there as well ofc.

Reply:
I totally forgot about it, thank you!
Now I added physics rotation as well.